app/components/user-menu.hbs

```diff
@@ -24,20 +24,20 @@
     <ClickOutside @onClickOutside={{fn (set this.isOpen) false}}>
       <ul class="menu">
         <li tabindex="-1" data-test-item>
-          <LinkTo @route="myprofile" @invokeAction={{set this.isOpen false}}>
+          <LinkTo @route="myprofile" {{on "click" (set this.isOpen false)}}>
```

Comment:
Unfortunately this doesn't work see https://stackoverflow.com/questions/69021116/add-extra-actions-to-linkto-in-octane I have an idea for a component to solve this problem, but this should be reverted here.

Reply:
:heavy_check_mark: